src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/iOS/PopupRenderer.ios.cs

```diff
@@ -148,25 +148,27 @@ void SetLayout()
 			{
 				var originY = Element.VerticalOptions.Alignment switch
 				{
-					LayoutAlignment.End => UIScreen.MainScreen.Bounds.Height - PreferredContentSize.Height,
-					LayoutAlignment.Center => (UIScreen.MainScreen.Bounds.Height / 2) - (PreferredContentSize.Height / 2),
+					LayoutAlignment.End => UIScreen.MainScreen.Bounds.Height,
+					LayoutAlignment.Center => UIScreen.MainScreen.Bounds.Height * 0.5,
 					_ => 0f
 				};
 
 				var originX = Element.HorizontalOptions.Alignment switch
 				{
 					LayoutAlignment.End => UIScreen.MainScreen.Bounds.Width,
-					LayoutAlignment.Center => UIScreen.MainScreen.Bounds.Width / 2,
+					LayoutAlignment.Center => UIScreen.MainScreen.Bounds.Width * 0.5,
 					_ => 0f
 				};
 
 				PopoverPresentationController.SourceRect = new CGRect(originX, originY, 0, 0);
+				PopoverPresentationController.PermittedArrowDirections = 0;
 			}
 			else
 			{
 				var view = Platform.GetRenderer(Element.Anchor).NativeView;
 				PopoverPresentationController.SourceView = view;
 				PopoverPresentationController.SourceRect = view.Bounds;
+				PopoverPresentationController.PermittedArrowDirections = UIPopoverArrowDirection.Up;
```

Comment:
Does this statement place the up arrow at the anchored element? I think if we can remove the arrows we should remove them across the board. Any arrow implementation should be a platform specific attached property since it isn't implemented in android or iOS

Reply:
Changed to use a platform specific.